microsoft/PowerToys
# Dependencies Cleanup

## Summary of the Pull Request

Removes obsolete dependencies from solution and setup.
PR touch multiple utilities since there are common projects but let me if is better to split this in multiple PRs.
I think it would be easy to review in a single PR. Early in the 0.65 cycle would be perfect! 😃 


## PR Checklist

- [x] **Closes:** #20129


## Detailed Description of the Pull Request / Additional comments

Removed obsolete libraries (old .NET Framework) and unused libraries:
- `System.Text.Json.dll` not needed on .NET 6: impacting multiple utilities with `Settings.UI.Library` as dependency.
- Markdown Preview
  - `HtmlAgilityPack.dll` not used
- PowerAccent
  - `GongSolutions.WPF.DragDrop.dll` not used
- Launcher
  - `Mono.Cecil.dll` (from wox porting) not used
  - `Mono.Cecil.Mdb.dll` (from wox porting) not used
  - `Mono.Cecil.Pdb.dll` (from wox porting) not used
  - `Mono.Cecil.Rocks.dll` (from wox porting) not used
  - `JetBrains.Annotations.dll` (resharper stuffs from wox porting) I guess this is not used since is outdated and installed only on some Launcher plugins
  - `vcamp140_app.dll` Visual C++ libraries now handled by the installer


## Validation Steps Performed

- Test passing
- Build setup
- Make sure the following utilities are working:
  - Fancy Zones Editor
  - Image Resizer
  - Preview
  - Settings
  - Hosts
  - PowerAccent
  - ColorPicker
  - PowerOCR
  - Awake
  - Launcher
